doc_developer/how_to_run_benchmarks.rst

```diff
@@ -0,0 +1,63 @@
+=====================
+How to run benchmarks
+=====================
+
+NumPy-Financial implements a benchmarking suite using
+`air speed velocity (asv) <https://asv.readthedocs.io/en/latest/>`_. This document
+provides a summary on how to perform some common tasks with asv.
+
+
+Getting asv
+===========
+
+asv is installed by using the ``lint`` dependency group of poetry. Make sure that you
+install the ``lint`` group before following the remaining steps.
+
+.. code-block::
+  :caption: Installing the lint dependency group
+
+      poetry install --with lint
```

Comment:
I'm not sure if these code blocks are rendered properly, will need to confirm before merging